reddit/uploads/api/views.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from users.models import User
+from serializers import PostSerializer, UserPostsSerializer
+
+
+class UserPostsView(generics.ListAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserPostsSerializer
+
+    def list(self, request, *args, **kwargs):
+        queryset = self.get_queryset()
+        serializer = self.get_serializer(queryset, many=True)
+        response = Response(serializer.data)
+        return response
+
+
+class PostCreateView(generics.CreateAPIView):
+    serializer_class = PostSerializer
+    permission_classes = (IsAuthenticated, )
+
+    def create(self, request, *args, **kwargs):
```

Comment:
The create method doesn't have any custom implementation. Why override the create method?